src/klein/resource.py

```diff
@@ -56,6 +60,5 @@ def __repr__(self):
         return "<special bound method/module klein.resource>"
 
 
-preserve = modules[__name__]
-modules[__name__] = _SpecialModuleObject()  # type: ignore
-modules[__name__].__preserve__ = preserve  # type: ignore
+module = _SpecialModuleObject(modules[__name__])
+modules[__name__] = module  # type: ignore[assignment]
```

Comment:
Assignment to the `modules` dict is apparently not cool according to `mypy` here because `_SpecialModuleObject` is not a `Module` object, as expected.

We ignore this because this whole thing is an unfortunate hack.